components/programs/ClassInstanceData.tsx

```diff
@@ -130,7 +137,7 @@ export interface ClassInstance extends MeetInstance {
   type: "class";
   startDate: string;
   endDate: string;
-  classDates: ClassDates;
+  classDates: ClassDates | ClassSDRDates;
```

Comment:
Since we only use `registrationDate` in either case we could use this instead:

```typescript
  classDates: Pick<ClassDates, "registrationDate">;
```